.sauce/cypress.yml

```diff
@@ -21,14 +21,18 @@ docker:
 #  image:
 #    name: saucelabs/stt-cypress-mocha-node
 #    tag: v0.3.3
+
 cypress:
   version: 5.6.0
   configFile: "tests/e2e/cypress.json"  # We determine related files based on the location of the config file.
 suites:
   - name: "saucy test"
     browser: "chrome"
     platformName: "Windows 10"
+    screenResolution: "2560x1600"  # Available resolutions: '800x600', '1024x768', '1152x864', '1280x768', '1280x800', '1280x960', '1280x1024', '1400x1050', '1440x900', '1600x1200', '1680x1050', '1920x1080', '1920x1200', '2560x1600'
```

Comment:
Note that resolutions are based on the platform, meaning that across platforms you could have different resolutions.  Maybe we want to update the comment saying `# Available resolutions for Windows 10`?

Reply:
sure! Updating.